package.json

```diff
@@ -41,7 +44,9 @@
     "space-separated-tokens": "^1.0.0"
   },
   "devDependencies": {
+    "@types/hast": "^2.3.1",
```

Comment:
I believe we’re putting dependent types in `dependencies` directly now, not in dev-deps.